test/test_sparse_csr.py

```diff
@@ -3361,16 +3338,28 @@ class TestSparseCompressedTritonKernels(TestCase):
     @dtypesIfCUDA(torch.half, *[torch.bfloat16] if SM80OrLater else [], torch.float)
     @unittest.skipIf(IS_FBCODE and IS_REMOTE_GPU, "Test requires Triton")
     def test_triton_bsr_dense_bmm(self, device, dtype, index_dtype, block_size):
+        import sys
+
+        if torch._running_with_deploy():
+            self.skipTest("Skipped for torch_deploy")
+
         from functools import partial
         from torch.sparse._triton_ops import bsr_dense_mm
 
-        kernel = _TritonLibrary.probablyRegisterOp(
+        def kernel_impl(*args, **kwargs):
+            return bsr_dense_mm(*args, skip_checks=True, **kwargs)
+
+        kernel = torch._TritonLibrary.registerOp(
```

Comment:
Shouldn't this be removed now that it is done during cuda init?

Reply:
It is needed for the tests below:
- we check that it is not possible to overwrite an already registered kernel.
- we check that the kernel was actually invoked